s3-import-commons/src/test/java/no/unit/nva/publication/s3imports/FilenameEventEmitterTest.java

```diff
@@ -54,6 +54,7 @@ class FilenameEventEmitterTest {
     
     public static final String EMPTY_SUBTOPIC = null;
     private static final String SOME_BUCKET = "someBucket";
+    private static final String VALID_SUBTOPIC_CRISTIN_IMPORT = "PublicationService.CristinData.DataEntry";
```

Comment:
Hvorfor ikke låne variabelen fra FilenameEventEmitter klassen. Da det er ett av de to mulige gyldige subtopics, hvis jeg forstår det rett